### PR TITLE
[master] Skip "PASSED"

### PR DESCRIPTION
**Pull request by leonkozlowski**
*NEW*
-Flip cli argument for `-s` to skip
-Default CLI to show all rule output

### CODE REVIEW

File: jpl/cli/__init__.py [hunk 1]
```diff
@@ -1 +1 @@
-from .main_cli import *
\ No newline at end of file
+from .main_cli import *
```
Comment (mitchbregs) [inline jpl/cli/__init__.py]: Yeah?

File: jpl/utils/__init__.py [hunk 2]
```diff
@@ -1 +1 @@
-from .path import *
\ No newline at end of file
+from .path import *
```
Comment (mitchbregs) [inline jpl/utils/__init__.py]: Same here

File: setup.py [hunk 3]
```diff
@@ -9,7 +9,7 @@
 
 setup(
     name="jpl",
-    version="0.0.2",
+    version="0.0.3",
```
Comment (mitchbregs) [inline setup.py]: Throw a `version.py` file containing `__version__='0.0.3'` maybe?

**Review by mitchbregs — APPROVED** [review]

File: jpl/cli/__init__.py (hunk 1 above)
Comment (leonkozlowski) [reply jpl/cli/__init__.py]: What do you suggest in place, this persists cli to user, bespoke imports?

Comment (mitchbregs) [reply jpl/cli/__init__.py]: What do you mean by that?  What's the need for this import here?  Generally just not a fan of importing `*`.

Comment (leonkozlowski) [reply jpl/cli/__init__.py]: In order to access the CLI, the functions need to be exposed up the call chain thru the `__init__.py`

Comment (mitchbregs) [reply jpl/cli/__init__.py]: Interesting...

**Comment by mitchbregs** [comment]
+1

File: jpl/utils/__init__.py (hunk 2 above)
Comment (leonkozlowski) [reply jpl/utils/__init__.py]: Resolving with abs imports

File: jpl/cli/__init__.py (hunk 1 above)
Comment (leonkozlowski) [reply jpl/cli/__init__.py]: Resolving with abs imports

**Review by mitchbregs — APPROVED** [review]
LGTM

**Pull request merged**